.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,105 @@
+name: Build, Lint, and Test
+
+on:
```

Comment:
It looks like you've copied these workflows directly from the module template. That makes sense given the original comment in #33 but I wonder if we ought to align even more closely, to prevent us from having to make further changes.

Can you copy [`main.yml` from the module template](https://github.com/MetaMask/metamask-module-template/blob/main/.github/workflows/main.yml) into this repo? That workflow ends up spawning `build-lint-and-test` and `publish-release`. Then you can re-copy `build-lint-test.yml` and `publish-release.yml` into this repo (specifically, we want the top here to say `on: workflow_call:` rather than `on: push:` or `on: pull_request`.

Reply:
@mcmire Thanks for your time and suggestion.
I've made this  as a reusable workflow.